TuneUp/ProfiledNodeViewModel.cs

```diff
@@ -130,47 +130,32 @@ public int? GroupExecutionOrderNumber
         private int? groupExecutionOrderNumber;
 
         /// <summary>
-        /// The most recent execution time of this node
+        /// The most recent execution time of this node in milliseconds
         /// </summary>
-        public TimeSpan ExecutionTime
+        public int ExecutionMilliseconds
         {
-            get => executionTime;
+            get => executionMilliseconds;
             set
             {
-                executionTime = value;
-                RaisePropertyChanged(nameof(ExecutionTime));
+                executionMilliseconds = value;
                 RaisePropertyChanged(nameof(ExecutionMilliseconds));
             }
         }
-        private TimeSpan executionTime;
-
-        /// <summary>
-        /// The total execution time of all node in the group.
-        /// </summary>
-        public TimeSpan GroupExecutionTime
-        {
-            get => groupExecutionTime;
-            set
-            {
-                groupExecutionTime = value;
-                RaisePropertyChanged(nameof(GroupExecutionTime));
-            }
-        }
-        private TimeSpan groupExecutionTime;
+        private int executionMilliseconds;
 
         /// <summary>
-        /// The most recent execution time of this node in milliseconds
+        /// The total execution time of the group in milliseconds
         /// </summary>
-        public int ExecutionMilliseconds
+        public int GroupExecutionMilliseconds
```

Comment:
This too.

Reply:
Thanks, I think this should be okay now.